runtime/moonbeam/src/lib.rs

```diff
@@ -21,8 +21,8 @@
 //! * Moonbeam tokenomics
 
 #![cfg_attr(not(feature = "std"), no_std)]
-// `construct_runtime!` does a lot of recursion and requires us to increase the limit to 256.
-#![recursion_limit = "256"]
+// `construct_runtime!` does a lot of recursion and requires us to increase the limit to 512.
+#![recursion_limit = "512"]
```

Comment:
Why did we change from 256?

Reply:
Because we exceed 256 recursions with the new pallet (you got an explicit compile error about that if you go back to 256)